src/App.js

```diff
@@ -1,12 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃

Reply:
This change has been done well